packages/cli/src/dev.ts

```diff
@@ -2,7 +2,7 @@ import { generateGlobalStyles } from "./commands/styles/lib/radius-styles";
 import 'dotenv/config';
 
 // To run in debug mode, add figma file url (located in package.json scripts -> cli: styles)
-const url = '';
+const url = 'https://www.figma.com/file/TJzz7ZB6pJvpLhjI5DWG3F/Radius-Design-Kit-V2(WIP)?node-id=725%3A8261';
```

Comment:
can we define this in an `env` file if it is not changing frequently or unless it is a test?

Reply:
I'll move this to .env